src/app/editor/shared/services/schema-validation.service.spec.ts

```diff
@@ -20,30 +20,29 @@
  * as an Intergovernmental Organization or submit itself to any jurisdiction.
 */
 
-import { beforeEachProviders, inject, it } from '@angular/core/testing';
+import { addProviders, inject } from '@angular/core/testing';
 
 import { SchemaValidationService } from './schema-validation.service';
 
 describe('SchemaValidationService', () => {
 
   const schema = { 'type': 'string', 'pattern': '[0-9]' };
 
-  beforeEachProviders(() => [SchemaValidationService]);
+  let service: SchemaValidationService;
 
-  it('should validate pattern for string value',
-    inject([SchemaValidationService],
-      (schemaValidationService: SchemaValidationService) => {
-        let value = '1';
-        expect(() => schemaValidationService.validateStringValue(value, schema)).not.toThrowError();
-      }
-    ));
+  beforeEach(() => addProviders([SchemaValidationService]));
 
-  it('should not validate pattern for string value',
-    inject([SchemaValidationService],
-      (schemaValidationService: SchemaValidationService) => {
-        let value = 'a';
-        expect(() => schemaValidationService.validateStringValue(value, schema)).toThrowError();
-      }
-    ));
+  beforeEach(inject([SchemaValidationService], (schemaValidationService) => {
+    service = schemaValidationService;
+  }));
 
+  it('should validate pattern for string value', () => {
+    let value = '1';
+    expect(() => service.validateStringValue(value, schema)).not.toThrowError();
+  });
+
+  it('should not validate pattern for string value', () => {
```

Comment:
Actually in both tests you validate the pattern right? You could actually move the two assertions into the first test/